test/fixtures/votes.yml

```diff
@@ -0,0 +1,19 @@
+vote1:
+  user_id: 10
```

Comment:
Instead of connecting your fixtures through IDs, use the name of your fixtures. You do not, (and should not), assign IDs in your fixtures.

```
vote1:
    user: ada
```